### PR TITLE
#104 migrate autoscaling api version from deprecated

### PR DESCRIPTION
**Pull request by razorman8669**
#104

### CODE REVIEW

File: charts/chatwoot/templates/web-hpa.yaml [hunk 1]
```diff
@@ -1,5 +1,5 @@
 {{- if .Values.web.hpa.enabled -}}
-apiVersion: autoscaling/v2beta2
+apiVersion: autoscaling/v2
```
Comment (vishnu-narayanan) [inline charts/chatwoot/templates/web-hpa.yaml:2]:
```suggestion
apiVersion: {{ .Values.autoscaling.apiVersion }}
```

File: charts/chatwoot/templates/worker-hpa.yaml [hunk 2]
```diff
@@ -1,5 +1,5 @@
 {{- if .Values.worker.hpa.enabled -}}
-apiVersion: autoscaling/v2beta2
+apiVersion: autoscaling/v2
```
Comment (vishnu-narayanan) [inline charts/chatwoot/templates/worker-hpa.yaml:2]:
```suggestion
apiVersion: {{ .Values.autoscaling.apiVersion }}
```

**Review by vishnu-narayanan — CHANGES_REQUESTED** [review]
Hey @razorman8669, Thank you for the PR.

Could you please make the following changes?

1. Make the `apiVersion` configurable so that older k8s installations are not affected. Create a new value `autoscaling.apiVersion` with default value `autoscaling/v2`. 
2. Increment the Chart version

**Comment by razorman8669** [comment]
@vishnu-narayanan I made the suggested changes.  I bumped the minor version since this is technically a breaking change for users of older kubernetes versions (pre v1.23)

**Review by vishnu-narayanan — APPROVED** [review]

**Comment by vishnu-narayanan** [comment]
@razorman8669 Thank you for the changes 🙇

**Pull request merged**